doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -282,6 +270,96 @@ Error list (HTTP response code in paranthesis):
 * UnsupportedMediaType (415). Request content type is not supported.
 * InvalidModification (422). Some piece of information is missing in payload.
 
+## Geospatial properties of entities
+
+The geospatial properties of a context entity can be represented by means of regular context attributes.
+The provision of geospatial properties enables the resolution of geographical queries.
+
+Two different syntaxes must be supported by compliant implementations: 
+
+* *Simple Location Format*. It is meant as a very lightweight format that developers and users can quickly
+and easily add to their existing entities with little effort.
+
+* *GeoJSON*.  [GeoJSON](https://tools.ietf.org/html/draft-butler-geojson-06)
+is a geospatial data interchange format based on the JavaScript Object Notation (JSON). GeoJSON provides greater flexibility
+allowing the representation of complex geospatial features.
+
+Additionally, geospatial properties might be declared by means of [civic addresses](https://schema.org/PostalAddress),
+but in that particular case entity providers will have to implement their own geocoding procedures,
+so that geographical queries can be resolved. 
+
+### Simple Location Format
+
+Simple Location Format supports basic geometries (*point* , *line* , *box* , *polygon*) and covers the typical
+use cases when encoding geographical locations. It has been inspired by [GeoRSS](http://www.georss.org/simple.html).
+
+It is noteworthy that the Simple Location Format is not intended to represent complex positions on Earth surface.
+For instance, those applications which require to capture altitude coordinates will have to use GeoJSON as
+representation format for the geospatial properties of their entities. 
+
+A context attribute  which represents a location encoded using the Simple Location Format
+must conform to the following syntax:
+
+* The attribute type must be one of the following values: (`geo:point`, `geo:line`, `geo:box` or `geo:polygon`).
+* The attribute value must be a list of coordinates, using the *WGS84* datum,
+with latitude and longitude units of decimal degrees. Such coordinate list will allow to encode the geometry
+specified by the `type` attribute and will be represented according to the specific rules defined below:
+
+  * Type `geo:point`:   the attribute value must contain a string containing a valid latitude-longitude pair, separated by a comma. 
+  * Type `geo:line`:    the attribute value must contain a string array of valid latitude-longitude pairs. There must be at least two pairs.
+  * Type `geo:polygon`: the attribute value must contain a string array of valid latitude-longitude pairs.
```

Comment:
Typically, implementations will require that polygon vertices be ordered to avoid "crossings" (e.g. [0,0], [0,2], [2,0], [2, 2] involves a cross). Thus I would suggest to specify that the verices should be in an order that avoid this kind of problems.

On the contrary, there is no need to have the last point be the same that the first one. It is an information easy to "complete" in the case the implementation needs it (as in the case of Orion in NGSIv1, where the polygon specificaiton in queries doesn't need the last point be the same that the first one but MongoDB does need, so Orion "complete" the information before interacting with MongoDB).